client/src/partials/projects/projects.html

```diff
@@ -44,120 +40,188 @@
+          <div class="panel-heading">{{ 'PROJECT.NEW' | translate }}</div>
+          <div class="panel-body">
+            <form class="form-horizontal" name="CreateForm" ng-submit="ProjectCtrl.submit(CreateForm.$invalid)" novalidate>
+
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.name.$invalid }">
+                <label class="col-xs-3 control-label required" for="bhima-project-name">{{ 'COLUMNS.NAME' | translate }}</label>
+                <div class="col-xs-9">
+                  <input type="text" class="form-control" name="name" id="bhima-project-name" ng-model="ProjectCtrl.project.name" required>
+                </div>
+              </div>
+
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.abbr.$invalid }">
+                <label for="bhima-project-abbr" class="col-md-3 control-label">{{ "COLUMNS.ABBR" | translate }}</label>
+                <div class="col-md-9">
+                  <input type="text" class="form-control" name="abbr" required id="bhima-project-abbr" ng-model="ProjectCtrl.project.abbr">
+                </div>
+              </div>
+
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.enterprise_id.$invalid }">
+                <label class="col-md-3 control-label">{{ "COLUMNS.ENTERPRISE" | translate }}</label>
+                <div class="col-md-9">
+                  <select class="form-control" name="enterprise_id" ng-model="ProjectCtrl.project.enterprise_id" id="bhima-enterprise-enterprise_id" ng-options="e.id as e.name for e in ProjectCtrl.enterprises | orderBy:'name'" required>
+                    <option value="" disabled> -- {{ 'SELECT.ENTERPRISE' | translate }} -- </option>
+                  </select>
+                </div>
+              </div>
+
+              <!-- zone de sante for snis -->
+              <div class="form-group" ng-class="{ 'has-error' : CreateForm.$submitted && CreateForm.zs_id.$invalid }">
+                <label class="col-md-3 control-label">{{ 'COLUMNS.ZS' | translate }}</label>
+                <div class="col-md-9">
+                  <select class="form-control" name="zs_id" ng-model="ProjectCtrl.project.zs_id" id="bhima-project-zs" ng-options="zs.id as zs.format for zs in ProjectCtrl.zones | orderBy:'zone'" required>
+                    <option value="" disabled> -- {{ 'SELECT.ZS' | translate }} -- </option>
+                  </select>
+                </div>
+              </div>
+              <!-- end zone de sante for snis -->
+              <div class="form-group">
+                <label class="col-md-3 control-label">{{ 'COLUMNS.LOCKED' | translate }}</label>
+                <div class="col-md-9">                
+                  <input type="checkbox" id="locked" ng-true-value="1" ng-false-value="0" name="bhima-project-locked" ng-model="ProjectCtrl.project.locked">
```

Comment:
The proper way to make a bootstrap checkbox input is to use the "checkbox" class.  [Here is the documentation](http://getbootstrap.com/css/#forms-horizontal).  You should be using the "checkbox" class on a `<div>` and wrapping the `<label>` element around the checkbox.